File: backend/src/main/java/com/andrew/hdss/dtos/FormResponsePushDto.java
package com.andrew.hdss.dtos;

import com.andrew.hdss.models.enums.FormResponseStatus;

import java.time.Instant;
import java.time.LocalDateTime;

public record FormResponsePushDto(
        String clientId,
        String visitClientId,
        Long formId,
        Integer formVersion,
        FormResponseStatus status,
        LocalDateTime startedAt,
        LocalDateTime completedAt
) {}

File: android/app/src/main/java/com/andrew/hdss/data/dtos/FormResponsePushDto.kt
package com.andrew.hdss.data.dtos

import com.andrew.hdss.data.models.enums.FormResponseStatus
import kotlinx.serialization.Serializable

@Serializable
data class FormResponsePushDto(
    val clientId: String,
    val visitClientId: String,
    val formId: Long,
    val formVersion: Int,
    val status: FormResponseStatus,
    val startedAt: String,
    val completedAt: String
)
